doc/usersguide/codes/openmc.rst

```diff
@@ -0,0 +1,134 @@
+..  _OpenMC: https://mit-crpg.github.io/openmc
+
+**Note: OpenMC supports the recommended** :ref:`UWUW`
+
+|
+
+Code-Specific steps for OpenMC
+==================================
+
+There are two varieties of code-specific steps for OpenMC_:
+
+1.  Defining attributes of the geometry using Cubit/Trelis groups
+2.  Defining DAGMC runtime parameters using the OpenMC input files
+
+Geometry metadata
+~~~~~~~~~~~~~~~~~
+
+The DAGMC geometry file can be used to define material assignments, boundary
+conditions, and temperatures.
+
+Materials
+---------
+
+The generic workflow description includes details on :ref:`grouping-basics`, but
+a specific naming convention is required for OpenMC. To define materials,
+the material ID must be provided in the group name. The format for the group
+name is as follows: ``mat:[matid]``.
+
+``[matid]`` should be replaced by the material ID that will be specified in the
+``materials.xml`` file.
+
+For example, consider a problem where material 7 should be assigned to volumes 4
+through 18. The following command should be used to specify that information:
+::
+
+    CUBIT> group "mat:7" add vol 4 to 18
+
+All volumes must belong to a group, if you want volumes to be filled with vacuum
+then add them to a group called, "mat:Vacuum":
+::
+
+    CUBIT> group "mat:Vacuum" add vol 4 to 18
+
+Surface boundary conditions
+----------------------------
+
+The following boundary conditions available in OpenMC are supported in
+DAGMC. These are:
+
+- vacuum
+- reflective
+- transmission
+
+For example, suppose you want to specify that surfaces 10 and 11 should be
+reflecting surfaces. This command would achieve that:
+::
+
+    CUBIT> group "boundary:Reflecting" add surf 10 11
+
+**Note: surfaces without a specified boundary condition will be set to ``transmission``.**
```

Comment:
This line is not being formatted correctly. I propose

    set to** ``transmission`` **.**

Reply:
Nice catch. Thanks!